highfidelity/hifi
# Ik enabled fix

This put the ik back off by default in desktop mode.
Bug is here: https://highfidelity.manuscript.com/f/cases/21602/Ik-not-disabled-on-desktop

### Test Plan
1) start interface in desktop
2) strafe left and right, and jump
3) result the animations play normally, ie the legs aren't straight when strafing
4) put on the hmd, 
5) move your arms and bend your legs,
6) result: they arms and legs should be bending with the ik targets
7) go to menu/developer/avatar/ and uncheck enable inverse kinematics
8) expected result: only the idle animation is visible in the hmd. arms and legs no longer respond to the controllers
